### DISCUSSION TITLE
Added more information in javadocs

### DISCUSSION THREAD

**Discussion by ikfir** [PaperMC/Paper#11452] [Ideas]
### Is your feature request related to a problem?

not enough info in the javadocs

### Describe the solution you'd like.

Adding more information in the javadocs like `@ApiStatus.AvailableSince("1.20-R0.1-SNAPSHOT")`

### Describe alternatives you've considered.

none

### Other

Yes it may be useless because you should only look on your current version of paper api but I think it will be much nicer to know since which version of the api the feature as been created at

**Comment by electronicboy** [comment]
Going back and adding this is something that we will generally not do, however, somebody externally iirc did put the effort in, it's just not tenable to maintain such a patch in the current repo state

**Comment by Leguan16** [comment]
I mean the alternative would be to go to spigot and convince them to do it, then paper would only have to do it on API they added which would be much more maintainable. Generally I think (lynx talked about it someday) they might add it - once the repo restructure happened - to paper-added API classes.
